resources/views/partials/forms/edit/company-select.blade.php

```diff
@@ -4,7 +4,7 @@
     <div class="form-group">
     {{ Form::label($fieldname, $translated_name, array('class' => 'col-md-3 control-label')) }}
         <div class="col-md-6">
-            <select class="js-data-ajax"  disabled="true" data-endpoint="companies" data-placeholder="{{ trans('general.select_company') }}" name="{{ $fieldname }}" style="width: 100%" id="company_select" aria-label="{{ $fieldname }}">
+            <select class="js-data-ajax"  disabled="true" data-endpoint="companies" data-placeholder="{{ trans('general.select_company') }}" name="{{ $fieldname }}" style="width: 100%" id="company_select" aria-label="{{ $fieldname }}" {{$multiple='true' ? 'multiple': ''}}>
```

Comment:
FYI, we need to check that `$multiple` is set, and also in this code you're setting `$multiple` to true, instead of evaluating it (`=` vs `==`).

You'd want something more like `{{ (isset($multiple) && ($multiple=='true')) ? " multiple='true'" : '' }}`.

By setting it instead of evaluating it, it causes the multi select to fire *anywhere* we use the company select.

<img width="1672" alt="Screenshot 2023-05-10 at 1 13 52 PM" src="https://github.com/snipe/snipe-it/assets/197404/f8f4a9b1-93ea-4981-a7ef-e80574b71100">

I fixed this in #13007, if you want to see.